src/Extensions/Compatability/Logger/Levels.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
+{
+	/// <summary>
+	/// List of all available levels for logging
+	/// </summary>
+	public static class Levels
```

Comment:
Do we really need this? Can we just convert all of the old levels to the new ones? We can definitely move that to a separate DPK, but it seems like it would be the best thing to do

Reply:
Compatibility NuGet package should be used only during migration from the usage of static classes to DI, so it's intentionally using old names and method definitions to simplify build of old code with minimal changes until you will rewrite it.